app/controllers/bookings_controller.rb

```diff
@@ -4,6 +4,14 @@ class BookingsController < ApplicationController
 
   def index
     @bookings = Booking.by_bill_id params[:bill_id]
-    @bill = @bookings.first.bill
+    booking = @bookings.first
+    @bill = booking.bill
+    @room_options = option_room_empty(booking)
```

Comment:
Viết như này được hơm 
```suggestion
   @bill = Bill.find params[:bill_id]
    @bookings = @bill.bookings
    @room_options = option_room_empty @bookings.first
```